en/SUMMARY.md

```diff
@@ -29,21 +29,32 @@
   * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
-* [Tutorials](tutorials/tutorials.md)
-  * [Ground Control Station](qgc/README.md)
-  * [Writing an Application](tutorials/tutorial_hello_sky.md)
-  * [Video Streaming in QGC](qgc/video_streaming.md)
-  * [Long-distance Video Streaming](qgc/video_streaming_wifi_broadcast.md)
+* [Hardware](hardware/README.md)
+  * [Autopilot Hardware](flight_controller/README.md)
+  * [Airframes](airframes/README.md)
+    * [Airframes Reference](airframes/airframe_reference.md)
+    * [Adding a New Airframe](airframes/adding_a_new_frame.md)
+    * [Parrot Bebop](advanced/parrot_bebop.md)
+  * [Porting Guide](debug/porting-guide.md)
+  * [Data Links](data_links/README.md)
+    * [SiK Radio](data_links/sik_radio.md)
+    * [Telemetry Wifi](advanced/telemetry_wifi.md)
```

Comment:
[This](https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/advanced/telemetry_wifi.html) is the only developer/integration topic because it explains what PX4 does, and hence is useful for someone wanting to create their own product.

What else do we need to say here? Ie 
- what else would someone need to do in order to connect to PX4.
- do we only connect to SIK firmware?
- are we planning to allow other types of firmware, and what would we point people to in order to do this.

Reply:
- I'd mention mavlink and maybe refer to another page?
- As I wrote above: anything with UART or UDP can be used
- What do you mean with 'other types of firmware'?